docs/HowTo/authentication/userpwd-db.md

```diff
@@ -17,7 +17,7 @@ To enable authentication with the Klaw database, follow these steps:
     encryption key to encrypt your password before it is stored in the database. The encryption key is used to secure and
     protect your password.
 
-    klaw.jasypt.encryptor.secretkey=kw2021secretkey
+    Klaw.jasypt.encryptor.secretkey=kw2021secretkey
```

Comment:
Changing spelling of "Klaw" should not be part of this PR, since it's for the rule "Klaw Core", can you reverse this one?